src/client/components/Reviews.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState, useEffect, useContext } from "react";
+import { MealsContext } from "./MealsContext";
+
+function Reviews() {
```

Comment:
This is good, I would just challenge with asking if you want a component of all the meals or maybe see only the relevant ones - for a specific meal?  🤔